cloudsync/sync/manager.py

```diff
@@ -369,14 +369,22 @@ def sync(self, sync: SyncEntry) -> bool:  # pylint: disable=too-many-branches
                 else:
                     self.state.split(sync)
 
-            response = self.embrace_change(sync, side, other_side(side))
+            try:
+                response = self.embrace_change(sync, side, other_side(side))
```

Comment:
I feel like it's more cohesive to catch the Exception in the `do` loop where the rest of the exception handling is done

Reply:
yeah, I considered that, but that would change the code flow - here we are in a loop that processes both sides of a SyncEntry before returning, moving it up one level could result in one side is skipped.

Reply:
I take some of that back -- need to break out of that loop on all response codes -- but, since we want to treat this error as 'finished' we need to call `self.finished(side, sync)` and `side` is not known in the `do` loop.

Reply:
Got it makes sense